javascript/src/stable.ts

```diff
@@ -78,6 +78,8 @@ export type ChangeOptions<T> = {
   time?: number
   /** A callback which will be called to notify the caller of any changes to the document */
   patchCallback?: PatchCallback<T>
+  /** copy doc references **/
+  allowDocRef?: boolean
```

Comment:
This should basically never be used externally right? Is there a way to hide it from the API? Maybe using the `@internal` annotation?

Reply:
I dont see any reason it needs to be internal only.  If someone wants to copy a chunk of a document from one place to another or from one document to another there is now a way.  It's not the default because it's probably not the behavior people are expecting most of the time but I'm happy its possible now.